src/components/HamburgerDropdown/index.tsx

```diff
@@ -1,12 +1,13 @@
 import { Button, ClickAwayListener, Grow,  Link,  MenuItem, MenuList, Paper, Popper } from '@mui/material';
 import React, { useEffect, useRef, useState } from 'react';
 import MenuIcon from '@mui/icons-material/Menu';
+import { styles } from './styles'
 
 const HamburgerDropDown = (props: any) => {
   const [open, setOpen] = useState(false);
   const anchorRef = useRef<HTMLButtonElement>(null);
 
-  const { headerNavigation } = props;
+  const { HEADER_NAVIGATION } = props;
```

Comment:
This would not be a situation where you'd have all caps. If it's a destructed property value it should be lower camel.